lib/chat_web/live/main_live/index.html.heex

```diff
@@ -480,6 +480,7 @@
                         msg={msg}
                         peer={@peer}
                         timezone={@timezone}
+                        room_map={@room_map}
```

Comment:
will it work with `@room_map |> Map.keys()`?

Reply:
yes